### ISSUE TITLE
[Bug] => Error when using IP address

### ISSUE THREAD

**Issue by iDuronto** [0xInfection/XSRFProbe#28]
#### Describe the bug
The bug concerned here is when using IP address instead of a domain, the program won't start nd would bug out with the error `tld.exceptions.TldDomainNotFound`.

#### Command You Used
`python3 xsrfprobe.py -u http://192.168.43.182/dvwa/vulnerabilities/csrf -c "PHPSESSID=dece0c0"`

#### Full Stack Trace Error
```
Traceback (most recent call last):
  File "xsrfprobe.py", line 12, in <module>
    from core import main  # import stuff
  File "C:\Users\user\Videos\xsrfprobe\core\main.py", line 33, in <module>
    from core.options import *
  File "C:\Users\user\Videos\xsrfprobe\core\options.py", line 162, in <module>
    os.makedirs('output/'+tld.get_fld(config.SITE_URL))
  File "C:\Python37\lib\site-packages\tld\utils.py", line 387, in get_fld
    search_private=search_private
  File "C:\Python37\lib\site-packages\tld\utils.py", line 339, in process_url
    raise TldDomainNotFound(domain_name=domain_name)
tld.exceptions.TldDomainNotFound: Domain 192.168.43.182 didn't match any existing TLD name!
```
#### Potential cause or fix
Catching the exception and handling the error as an IP address would resolve the issue.

#### Environment:
- OS: Kali Rolling 2018.3
- Python version: v3.7
#### Some Questions

- [x] I am using the latest version of XSFRProbe.
- [x] I installed the dependecies using `pip3` instead of `pip`.
- [x] I have read the documentation before submitting this issue.
- [x] I have checked the other issues to see if someone reported this before.

#### Other stuff
[Something else you want the author to know?]

**Comment by 0xInfection** [comment]
Hey there, thanks for this issue. I have created [this PR](#29) which resolves your issue. Can you please confirm that the issue has been fixed in [Fixing-#29](https://github.com/0xInfection/XSRFProbe/tree/Fixing-%2328) branch?

**Comment by iDuronto** [comment]
Yeah I can confirm its has worked. Thank you!

You're fast as lightning.
